ducktape/services/service.py

```diff
@@ -150,6 +150,12 @@ def wait(self):
         """
         pass
 
+    def wait_node(self, node, timeout_sec=None):
```

Comment:
I have no strong preference, but it seemed reasonable to add it since we already expose node-specific methods in `Service`.  What do you think?